app/src/main/java/com/android/sample/ui/activity/CreateActivity.kt

```diff
@@ -324,29 +327,63 @@ fun CreateActivityScreen(
                         .align(Alignment.CenterHorizontally)
                         .fillMaxWidth()
                         .padding(STANDARD_PADDING.dp),
-                expanded = expanded,
-                onExpandedChange = { expanded = !expanded }) {
+                expanded = expandedType,
+                onExpandedChange = { expandedType = !expandedType }) {
                   OutlinedTextField(
                       readOnly = true,
-                      value = selectedOption,
+                      value = selectedOptionType,
                       onValueChange = {},
                       label = { Text("Activity Type") },
                       trailingIcon = {
-                        ExposedDropdownMenuDefaults.TrailingIcon(expanded = expanded)
+                        ExposedDropdownMenuDefaults.TrailingIcon(expanded = expandedType)
                       },
                       colors = ExposedDropdownMenuDefaults.textFieldColors(),
-                      modifier = Modifier.menuAnchor().fillMaxWidth())
+                      modifier = Modifier.menuAnchor().fillMaxWidth().testTag("typeTextField"))
                   ExposedDropdownMenu(
-                      expanded = expanded,
-                      onDismissRequest = { expanded = false },
+                      expanded = expandedType,
+                      onDismissRequest = { expandedType = false },
                       modifier = Modifier.fillMaxWidth().padding(STANDARD_PADDING.dp)) {
```

Comment:
Great that you are clarifying the expanded corresponding to the Type, amongst the other expanded we use.